addresses.go

```diff
@@ -16,12 +16,10 @@ type Address struct {
 	CountryCode string
 	// FirstName - the first name of the person associated with the address
 	FirstName string
+	// ID - the ID of the address
+	ID uint64
 	// LastName - the last name of the person associated with the address
 	LastName string
-	// Latitude - the latitude of the address
```

Comment:
Are you removing these because they do not exist on an address? So they are not on a customer BUT they are on an order address. So removing this will no doubt break code that's using the package and relying on these tags. 

It looks like historically we were using the same struct for both customer addresses and orders but they are slightly different possibly it seems.

Reply:
BILLING ADDRESS/SHIPPING ADDRESS

address1: The street address of the billing address.
address2: An optional additional field for the street address of the billing address.
city: The city, town, or village of the billing address.
company: The company of the person associated with the billing address.
country: The name of the country of the billing address.
country_code: The two-letter code (ISO 3166-1 format) for the country of the billing address.
first_name: The first name of the person associated with the payment method.
last_name: The last name of the person associated with the payment method.
latitude: The latitude of the billing address.
longitude: The longitude of the billing address.
name: The full name of the person associated with the payment method.
phone: The phone number at the billing address.
province: The name of the region (for example, province, state, or prefecture) of the billing address.
province_code: The two-letter abbreviation of the region of the billing address.
zip: The postal code (for example, zip, postcode, or Eircode) of the billing address.

CUSTOMER ADDRESS

address1: The customer's mailing address.
address2: An additional field for the customer's mailing address.
city: The customer's city, town, or village.
company: The customer's company.
country: The customer's country.
country_code: The two-letter country code corresponding to the customer's country.
country_name: The customer's normalized country name.
customer_id: A unique identifier for the customer.
default: Whether this address is the default address for the customer.
first_name: The customer's first name.
id: A unique identifier for the address.
last_name: The customer's last name.
name: The customer's first and last names.
phone: The customer's phone number at this address.
province: The customer's region name. Typically a province, a state, or a prefecture.
province_code: The code for the region of the address, such as the province, state, or district. For example QC for Quebec, Canada.
zip: The customer's postal code, also known as zip, postcode, Eircode, etc.

These are the differences. Looks like when this was created it was an order one as it had the latitude and longitude. If you check the struct it matches EXACTLY against the BILLING/SHIPPING address of an order and is only used by the order repository.

I think it would be best to create a whole new CustomerAddress struct for customers as there is a light difference and the models match.

Reply:
Oh yep, you're right! I didn't know the `Order` object had these fields 😁 I thought they were obsolete

I'll add them back 🙌

Reply:
Should be fixed now - I made changes to `httpshopify` and `lod-patient-api` for this 😄